contribute_to_pim/front.rst

```diff
@@ -0,0 +1,152 @@
+
+The .less files are compiled into a main css file with lessphp.
+
+This command also compiles the old requirejs configuration and translation files into javascript code later injected in the frontend modules.
+
+Run webpack
++++++++++++
+
+The last step is now to run the webpack compilation step to generate the final javascript artefacts.
+
+Sum up
+++++++
+
+With those steps, we just prepared everything to get started to contribute to the frontend part of the application. You are now ready to launch the application and start to experiment with it.
+
+Quick FAQ to understand what is going on
+----------------------------------------
+
+.. note::
+
+    This section is here to understand how the Akeneo PIM front commands are working, if you want to be quick and efficient go to the QuickAndEasyWay_ section.
+
+What to do if I updated a translation?
+++++++++++++++++++++++++++++++++++++++
+
+To rebuild the frontend translations you need to run:
+
+.. code-block:: bash
+
+    rm -rf var/cache/*
+    rm -rf web/js/translation/your_updated_locale.js
+    bin/console --env=prod oro:translation:dump your_updated_locale
+
+What to do if I updated a .less/.css file?
+++++++++++++++++++++++++++++++++++++++++++
+
+.. code-block:: bash
+
+    bin/console --env=prod oro:assetic:dump
+
+What to do if I updated a form_extension**.yml file?
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+.. code-block:: bash
+
+    rm -rf var/cache
+    bin/console --env=prod pim:installer:dump-extensions
+    yarn webpack-dev
+
+What to do if I updated a requirejs.yml file?
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+.. code-block:: bash
+
+    rm -rf var/cache
+    bin/console --env=prod pim:installer:dump-require-paths
+    yarn webpack-dev
+
+Conclusion
+++++++++++
+
+With those commands you now know what to do exactly when you modify some files on the PIM. But most of the time there are more efficient way to do.
+
+.. _QuickAndEasyWay:
+
+The quick and easy way
+----------------------
+
+Most of the time, when you are contributing to the PIM you do a bit of everything at the same time. Sometimes, it can be hard to keep track of which command to run and when. That's why it can be really handy to define an alias in your terminal to run a compilation of them.
+
+If you touch a configuration file, a .less file or a translation file (anything but a javascript file)
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+run this command:
+
+.. code-block:: bash
+
+    rm -rf ./var/cache/*; rm -rf ./web/js/*; rm -rf ./web/css/*; /usr/local/opt/php\@7.2/bin/php bin/console pim:installer:dump-extensions --env=prod; /usr/local/opt/php\@7.2/bin/php bin/console pim:install:ass --env=prod --symlink;
```

Comment:
The path is for mac only, you can jut put `php` instead ;)

Reply:
ha yeah, good catch